data/wanderer ships.txt

```diff
@@ -211,11 +211,14 @@ ship "Strong Wind"
 		"mass" 260
 		"drag" 4.7
 		"heat dissipation" .7
-		"fuel capacity" 400
+		"fuel capacity" 600
 		"cargo space" 68
-		"outfit space" 493
+		"outfit space" 496
```

Comment:
Remove a thunderhead storage array to add another ramscoop instead of increasing the outfit space.

Reply:
Done.